core/roboconf-agent/src/test/java/net/roboconf/agent/internal/misc/AgentUtilsTest.java

```diff
@@ -271,4 +279,32 @@ public void testFindIpAddress_nforceDefaultNetworkInterface() throws Exception {
 		String expectedIp = InetAddress.getLocalHost().getHostAddress();
 		Assert.assertEquals( expectedIp, AgentUtils.findIpAddress( AgentConstants.DEFAULT_NETWORK_INTERFACE ));
 	}
+
+	@Test
+	public void testExecuteScriptResources() throws IOException, InterruptedException {
+
+		//prepare our resources
+		File scriptsDir = this.folder.newFolder();
+
+		AgentUtils.executeScriptResources( scriptsDir );
+		Assert.assertEquals(0, scriptsDir.listFiles().length);
+
+		Assume.assumeTrue( isUnix());
+
+		File script1 = new File( scriptsDir, "toto-script-all.sh" );
+		File script2 = new File( scriptsDir, "titi-script.sh" );
+		Utils.writeStringInto( "#!/bin/bash\necho totototototo > toto.txt", script1);
```

Comment:
Once the main script is executed,  "toto.txt" is created in the same location than script files (scriptsDir a temporary folder)